src/assets/schemas/site-area/site-area.json

```diff
@@ -37,6 +37,11 @@
     "siteID": {
       "$ref": "common#/definitions/id"
     },
+    "parentSiteAreaID": {
+      "type": ["string", "null"],
+      "pattern": "^$|^[0-9a-fA-F]{24}$",
+      "sanitize": "mongo"
```

Comment:
Yes in fact it is a regular id, but it needs to be nullable, because otherwise we are not able to clear the site area parent again. 

I will change it to "nullable: true" instead of the array.